OpenRA.Mods.Common/Widgets/ViewportControllerWidget.cs

```diff
@@ -286,7 +286,8 @@ public override bool HandleMouseInput(MouseInput mi)
 
 			if (scrollType == MouseScrollType.Standard || scrollType == MouseScrollType.Inverted)
 			{
-				if (mi.Event == MouseInputEvent.Move)
+				if (mi.Event == MouseInputEvent.Move &&
+					(isStandardScrolling || (Viewport.LastMousePos - mi.Location).Length > Game.Settings.Game.MouseScrollDeadzone))
```

Comment:
This needs to be relative to the start of the move, not the last pos. If you move your mouse slow enough it will never trigger.